utils/manifests/generateComponent.go

```diff
@@ -0,0 +1,74 @@
+package manifests
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+)
+
+func generateComponents(manifest string, resource int, cfg Config) (*Component, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	var binPath string = wd + "/utils/manifests/utilBin/kubeopenapi-jsonschema"
+	switch runtime.GOOS {
+	case "windows":
+		{
+			binPath += ".exe"
+		}
+	case "darwin":
```

Comment:
Does this mean that Linux is the default or that Linux is not supported?

Reply:
@leecalcote  the binary for linux is kubeopenapi-jsonschema. If the platforms are darwin or windows, these postfixes are added because that is the name of their respective binaries. And if neither darwin nor windows is the platform, then it is assumed to be linux.